src/features/auth/keycloak.ts

```diff
@@ -8,10 +8,12 @@ export const kcInitOption: KeycloakInitOptions = {
 }
 
 // Let Typescript know we are using the 'native' promise type
-const instance = window.Keycloak<'native'>({
-  url: KC_AUTH_URL,
-  realm: KC_REALM,
-  clientId: KC_CLIENT
-})
+const instance = window.Keycloak
+  ? window.Keycloak<'native'>({
+      url: KC_AUTH_URL,
+      realm: KC_REALM,
+      clientId: KC_CLIENT
+    })
+  : null
```

Comment:
what effect does this change have?

Reply:
It prevents our app from crashing when`https://sso-dev.pathfinder.gov.bc.ca/auth/js/keycloak.js` is not available like last time when the platform service was upgrading their SSO

Reply:
ahhhhhh nice. Smart!